src/app/actions.js

```diff
@@ -107,7 +107,7 @@ export function logOut(store) {
   async function logOut(dispatch) {
     const state = store.getState();
     const identityProviderAvailable =
-      state?.root?.config?.identityProviderAvailable;
+      state?.general?.config?.identityProviderAvailable;
```

Comment:
I'd be happy with another name if you have any other suggestions @goulinkh?

Reply:
`general` seems fine to be, but you could also name it: `connection`, `config`,`login` based on the list of actions that the reducer handles.